### PR TITLE
[DLS-4524][Suresh]: Play 2.8 upgrade

### PR DESCRIPTION
**Pull request by sureshhmrc**

### CODE REVIEW

**Review by rajanimohan22 — APPROVED** [review]
Changes reviewed and approved.

File: project/plugins.sbt [hunk 1]
```diff
@@ -6,7 +8,7 @@ addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.0.0")
```
Comment (rambabu-posa) [inline project/plugins.sbt:8]: Did you try "3.4.0" or "3.5.0" versions for sbt auto build?

File: test/unit/controllers/CustomsDeclarationsMetricsControllerSpec.scala [hunk 2]
```diff
@@ -37,8 +36,7 @@ import util.TestData._
 import scala.concurrent.{ExecutionContext, Future}
 import scala.util.Try
 
-class CustomsDeclarationsMetricsControllerSpec extends UnitSpec
-  with Matchers with MockitoSugar {
+class CustomsDeclarationsMetricsControllerSpec extends UnitSpec with MockitoSugar {
```
Comment (rambabu-posa) [inline test/unit/controllers/CustomsDeclarationsMetricsControllerSpec.scala:39]: Duplicate `MockitoSugar` as its already coming from `UnitSpec`

File: project/plugins.sbt (hunk 1 above)
Comment (sureshhmrc) [reply project/plugins.sbt:8]: thats creating issues in this service with some other plugin , so not used the latest version of sbt-auto-build

File: test/unit/controllers/CustomsDeclarationsMetricsControllerSpec.scala (hunk 2 above)
Comment (sureshhmrc) [reply test/unit/controllers/CustomsDeclarationsMetricsControllerSpec.scala:39]: sure, will remove it

**Review by rambabu-posa — APPROVED** [review]

**Pull request merged**